packages/dcos-history/extra/history/server.py

```diff
@@ -90,4 +92,10 @@ def start():
 
     state_buffer = BufferCollection(os.environ['HISTORY_BUFFER_DIR'])
     BufferUpdater(state_buffer, headers_cb).run()
-    app.run(host='0.0.0.0', port=int(os.environ.get('PORT', '15055')))
+
+
+def start():
```

Comment:
If we need to explicitly invoke gunicorn from outside of python to get the expected behavior, then why leave this option around? I would just remove this entirely as well as the hook in setup.py